Kingfisher-Demo/ViewController.swift

```diff
@@ -57,12 +57,12 @@ extension ViewController: UICollectionViewDataSource {
     
     override func collectionView(collectionView: UICollectionView, cellForItemAtIndexPath indexPath: NSIndexPath) -> UICollectionViewCell {
         let cell = collectionView.dequeueReusableCellWithReuseIdentifier("collectionViewCell", forIndexPath: indexPath) as! CollectionViewCell
+        cell.cellImageView.kf_showIndicatorWhenLoading = true
```

Comment:
Why don't you add the activity indicator flag as a boolean option that's part of `KingfisherOptionsInfo`